CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+2.0.0
+-----
+
+* Added Enum type
+* Added OneOfValidator
+* **Backwards incompatible**: Boolean now nullable
```

Comment:
Explain what part about this is backwards incompatible

Reply:
I've updated the description and wording, its not as much backwards incompatible but more a potentially breaking change.